dvc/output/__init__.py

```diff
@@ -41,7 +41,8 @@
 # so when a few types of outputs share the same name, we only need
 # specify it once.
 CHECKSUM_SCHEMA = {
-    schema.Optional(RemoteLOCAL.PARAM_CHECKSUM): schema.Or(str, None),
+    schema.Optional(utils.CHECKSUM_MD5): schema.Or(str, None),
```

Comment:
should we move hashing related stuff to a separate module? so, that it looks like `hash.CHECKSUM_MD5`. `utils` should contain really generic stuff, everything else should be moving into packages with some meaningful names